tools/travis/stage.sh

```diff
@@ -213,6 +220,9 @@ function get_test_modules_for_stage() {
         (${STAGE_TESTS})
             echo "-pl $modules_tests"
         ;;
+        (${STAGE_CONNECTOR_HIVE_1})
+            echo "-pl $MODULES_CONNECTOR_HIVE -Phive-1.2.1 clean"
```

Comment:
nit: can you add a comment here to note that 'clean' will only recompile flink-connector-hive module?